### PR TITLE
Intermediate HTML and CSS && Intermediate CSS Concepts: Reformatted headers to follow sentence case

### PR DESCRIPTION
**Pull request by amc131**
## Because

Reformats the headers to be sentence case and free of code snippets per the style guide

## This PR

- Removed embedded code snippets from the headers
- Formatted the headers to follow sentence case

## Issue

Related to #25861 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by ManonLef** [comment]
Thanks so much! I noticed a few lessons from the [intermediate HTML Concepts](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css#intermediate-html-concepts) that still need updates are not included in your PR. Would you mind doing those before I review? 

It involves the `SVG` and `Tables` lessons.

**Comment by ManonLef** [comment]
Wow that was quick 😄 thanks so much! I'll review it now!

**Comment by amc131** [comment]
@ManonLef Everything was done on my end, I just forgot to push my final commit, sorry about that :)

File: intermediate_html_css/intermediate_css_concepts/browser_compatibility.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ In 1995 the world got introduced to the first version of Internet Explorer, whic
 
 You're most likely familiar with most, if not all these names.<span id="most-used-browser"> There is a lot of competition among the browsers still to this day, even though Chrome (and [Chromium](https://en.wikipedia.org/wiki/Chromium_(web_browser))) is the dominant player in the market</span>.
 
-### What is Browser Compatibility
+### What is browser compatibility
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/browser_compatibility.md:25]:
```suggestion
### What is browser compatibility?
```

File: intermediate_html_css/intermediate_css_concepts/browser_compatibility.md [hunk 2]
```diff
@@ -32,23 +32,23 @@ Because of these differences, your applications may behave differently in the br
...
 
 W3C [World Wide Web Consortium](https://www.w3.org/) is the authority behind developing web standards to maximize accessibility and consistency of the web experience. W3C is also the authority to develop new features in the CSS. This is a closely collaborative approach with the Web as a community as well as the companies developing the web browsers.
 
 When the web browsers such as Nexus and Netscape were released, there was no such organization as W3C to help create more compatibility. Your application could look and function differently in each browser. Even worse, your application could've been completely unusable. Web developers had to make specific adjustments for each browser, and not every developer had enough resources to make that work for everyone.
 
 Today, as the standards around the web evolve and change, and web developers begin implementing new features in their codebase, the browsers must provide support for those new features. If the user experience is impacted by the lack of support in the browsers, the users might find their way to the competitor.
 
-### When Is It Safe to Use New Features
+### When is it safe to use new features
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/browser_compatibility.md:43]:
```suggestion
### When is it safe to use new features?
```

File: intermediate_html_css/intermediate_css_concepts/css_units.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 ### Introduction
 There are many different units that you can use to define sizes in CSS. This lesson will introduce the most important to you, and show you where to learn about the rest of them.
 
-### Learning Outcomes
+### Learning outcomes
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/css_units.md:4]:
```suggestion
### Learning outcomes

```

File: intermediate_html_css/intermediate_css_concepts/css_units.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 * You'll learn the difference between relative and absolute units
 * You'll learn when it's appropriate to use the different units.
 
-### Absolute Units
+### Absolute units
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/css_units.md:8]:
```suggestion
### Absolute units

```

File: intermediate_html_css/intermediate_css_concepts/css_units.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 Absolute units are those that are always the same in any context. `px` is an absolute unit because the size of a pixel doesn't change relative to anything else on the page. In fact, `px` is the only absolute unit you should be using for web projects. The rest of them make more sense in a print setting because they are related to physical units such as `in` (inch) and `cm` (centimeter).
 
-### Relative Units
+### Relative units
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/css_units.md:11]:
```suggestion
### Relative units

```

File: intermediate_html_css/intermediate_css_concepts/css_units.md (hunk 3 above)
Comment (ManonLef) [reply intermediate_html_css/intermediate_css_concepts/css_units.md:4]: These headers seem to be missing a newline below them so I decided to fix them while I'm at it. Feel free to `commit suggestion` these if you approve

File: intermediate_html_css/intermediate_css_concepts/css_units.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
 Relative units are units that can change based on their context. There are several of them that you are likely to encounter and want to use.
 
 #### em and rem
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/css_units.md:14]:
```suggestion
#### em and rem

```

File: intermediate_html_css/intermediate_css_concepts/css_units.md [hunk 7]
```diff
@@ -31,14 +31,14 @@ The units `vh` and `vw` relate to the size of the viewport. Specifically, `1vh`
 3. [Fun with Viewport Units](https://css-tricks.com/fun-viewport-units/) demonstrates some interesting things you can do with `vh` and `vw`.
 </div>
 
-### Knowledge Check
+### Knowledge check
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/css_units.md:34]:
```suggestion
### Knowledge check

```

File: intermediate_html_css/intermediate_css_concepts/css_units.md [hunk 8]
```diff
@@ -31,14 +31,14 @@ The units `vh` and `vw` relate to the size of the viewport. Specifically, `1vh`
...
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
 - [Why would you want to use `em` or `rem` for font-size instead of `px`?](#em-and-rem)
 - [What are some instances where you might want to use `vh` and `vw`?](#viewport-units)
 - [What are some instances where you might want to use `px` instead of a relative unit?](https://codyloyd.com/2021/css-units/)
 
-### Additional Resources
+### Additional resources
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/css_units.md:41]:
```suggestion
### Additional resources

```

File: intermediate_html_css/intermediate_css_concepts/custom_properties.md [hunk 9]
```diff
@@ -138,15 +138,15 @@ Using the `prefers-color-scheme` media query can be pretty helpful for users sin
 3. Open the inspector on this page to inspect the styles and see how Odin uses some custom properties.
 </div>
 
-### Knowledge Check
+### Knowledge check
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/custom_properties.md:141]:
```suggestion
### Knowledge check

```

File: intermediate_html_css/intermediate_css_concepts/custom_properties.md [hunk 10]
```diff
@@ -138,15 +138,15 @@ Using the `prefers-color-scheme` media query can be pretty helpful for users sin
...
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
 * [How would you declare a custom property with a name of `text-color`?](#using-custom-properties)
 * [How would you access a custom property with a name of `background-color`?](#using-custom-properties)
 * [Where would you declare a custom property to have its scope be global and accessible by all other selectors?](#scope)
 * [Where would you declare a custom property so that a user's theme setting from their OS or browser was taken into account?](#creating-themes-with-custom-properties)
 
-### Additional Resources
+### Additional resources
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/custom_properties.md:149]:
```suggestion
### Additional resources

```

File: intermediate_html_css/intermediate_css_concepts/more_css_properties.md [hunk 11]
```diff
@@ -39,7 +39,7 @@ Opacity is another easy one that can be very useful in some circumstances.
 
 Check out [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity) for a definition and some examples.
 
-### Knowledge Check
+### Knowledge check
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_css_properties.md:42]:
```suggestion
### Knowledge check

```

File: intermediate_html_css/intermediate_css_concepts/more_css_properties.md [hunk 12]
```diff
@@ -49,7 +49,7 @@ This section contains questions for you to check your understanding of this less
 - [Which property would you use to create rounded corners on an element?](#borders)
 - [How would you use border-radius to make a circular element?](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius)
 
-### Additional Resources
+### Additional resources
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_css_properties.md:52]:
```suggestion
### Additional resources

```

File: intermediate_html_css/intermediate_css_concepts/more_css_properties.md [hunk 13]
```diff
@@ -3,7 +3,7 @@ By now, you probably have a strong grasp of the important foundational concepts
 
 There are a _lot_ of CSS properties. [A Recent CSS Tricks Article](https://css-tricks.com/how-many-css-properties-are-there/) says there are _hundreds_. Luckily, you don't have to memorize them all: the amount of properties you're actually going to use on a daily basis is much smaller. This lesson is going to cover most of the items you'll end up using on a regular basis. The format of this lesson is a little different since it's essentially just a list of CSS properties. We'll give a little description of the property and then link you to some documentation where you can see all the available options.
 
-### Learning Outcomes
+### Learning outcomes
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_css_properties.md:6]:
```suggestion
### Learning outcomes

```

File: intermediate_html_css/intermediate_css_concepts/more_css_properties.md [hunk 14]
```diff
@@ -3,7 +3,7 @@ By now, you probably have a strong grasp of the important foundational concepts
...
 - You'll learn about a lot of useful CSS properties!
 
 #### Background
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_css_properties.md:9]:
```suggestion
#### Background

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 15]
```diff
@@ -1,14 +1,14 @@
 ### Introduction
 You've already done a bit of text-manipulation in our Foundations lessons. This lesson will cover a few more useful CSS properties that can be used when working with text.
 
-### Learning Outcomes
+### Learning outcomes
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:4]:
```suggestion
### Learning outcomes

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 16]
```diff
@@ -1,14 +1,14 @@
 ### Introduction
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:1]:
```suggestion
### Introduction

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 17]
```diff
@@ -1,14 +1,14 @@
...
 You've already done a bit of text-manipulation in our Foundations lessons. This lesson will cover a few more useful CSS properties that can be used when working with text.
 
-### Learning Outcomes
+### Learning outcomes
 * You'll learn how to use custom fonts on your web projects.
 * You'll learn some more text-related CSS properties.
 
 ### Fonts
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:8]:
```suggestion
### Fonts

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 18]
```diff
@@ -1,14 +1,14 @@
...
 In our Foundations lesson, we covered changing the `font-family` of an element, but there is some nuance and detail that we left out at the time.
 
-#### The System Font Stack
+#### The system font stack
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:11]:
```suggestion
#### The system font stack

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 19]
```diff
@@ -21,7 +21,7 @@ body {
 
 The point of this somewhat ridiculous string of font-families is to try using the default font of the system's user interface. It will go through each of those fonts until it finds one that is installed on the system, and then use that. Using a stack like this often produces pleasing results, especially if you're going for a somewhat 'neutral' font style.
 
-#### Online Font Libraries
+#### Online font libraries
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:24]:
```suggestion
#### Online font libraries

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 20]
```diff
@@ -48,7 +48,7 @@ body {
 
 Keep in mind that it's important to add a fallback font.  If you're linking to an external API, you have no guarantee that the URL won't change, or that the external API won't go down at some point. Having a reasonable fallback means that if something goes wrong, at least your site won't look completely broken.
 
-#### Downloaded Fonts
+#### Downloaded fonts
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:51]:
```suggestion
#### Downloaded fonts

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 21]
```diff
@@ -64,7 +64,7 @@ h1 {
 
 This method _may_ be more reliable than relying on a third-party font API, but it is always wise to include a fallback.
 
-### Text Styles
+### Text styles
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:67]:
```suggestion
### Text styles

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 22]
```diff
@@ -64,7 +64,7 @@ h1 {
...
 You learned the basics of manipulating fonts in our Foundations lessons, but there is quite a bit more that you can do with CSS when it comes to manipulating text styles. These rules are all relatively simple and self-explanatory. You can refer to the docs for any questions you might have.
 
 #### font-style
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:70]:
```suggestion
#### font-style

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 23]
```diff
@@ -135,15 +135,15 @@ The full snippet is:
 
 You can see more detail and an example in [this CSS Tricks Article](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/).  (Be ready to go look that article up every time you want to use this.)
 
-### Knowledge Check
+### Knowledge check
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:138]:
```suggestion
### Knowledge check

```

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 24]
```diff
@@ -135,15 +135,15 @@ The full snippet is:
...
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
 - [What are the 2 ways to add fonts that are not installed on a user's computer?](#online-font-libraries)
 - [What is the 'system font stack' and why would you want to use it?](#the-system-font-stack)
 - [Which property would you use to increase or decrease the space between letters in a word?](#letter-spacing)
 - [Which property would you use to increase or decrease the space between lines in a paragraph?](#line-height)
 
-### Additional Resources
+### Additional resources
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:146]:
```suggestion
### Additional resources

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 25]
```diff
@@ -2,16 +2,16 @@
 
 By now you have had quite a bit of practice moving elements around the screen using things like margin, padding, and flexbox. These techniques have all relied on CSS's default "positioning-mode". This default positioning-mode is intuitive, and you'll continue using it for almost all of your layout needs. However, there are other methods at your disposal that can be very useful in some situations.
 
-### Learning Outcomes
+### Learning outcomes
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:5]:
```suggestion
### Learning outcomes

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 26]
```diff
@@ -2,16 +2,16 @@
...
  - You'll learn how to use absolute positioning.
  - You'll learn how to use fixed positioning.
  - You'll learn how to use sticky positioning.
  - You'll know the difference between each property and how to combine them.
 
-### Static and Relative Positioning
+### Static and relative positioning
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:11]:
```suggestion
### Static and relative positioning

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 27]
```diff
@@ -2,16 +2,16 @@
...
 The default positioning mode that you've gotten used to is ``position: static``. The difference between static and relative is fairly simple. Static is the default position of every element, and properties ``top``, ``right``, ``bottom``, and ``left`` do not affect the position of the element. Relative on the other hand is pretty much the same as static, but properties  ``top``, ``right...(etc.)`` displace the element relative to its normal position in the flow of the document.
 
-### Absolute Positioning
+### Absolute positioning
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:14]:
```suggestion
### Absolute positioning

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 28]
```diff
@@ -29,10 +29,10 @@ In the following example, we are using absolute positioning to display text over
 
 Disclaimer: absolute positioning has very specific use cases and if possible, using flexbox or grid should be prioritized. Absolute positioning shouldn't be used to do entire page layouts.
 
-### Fixed Positioning
+### Fixed positioning
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:32]:
```suggestion
### Fixed positioning

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 29]
```diff
@@ -29,10 +29,10 @@ In the following example, we are using absolute positioning to display text over
...
 Fixed elements are also removed from the normal flow of the document and are positioned relative to the ``viewport``. You basically use  ``top``, ``right``, ``bottom``, and ``left`` properties to position it, and it will stay there as the user scrolls. This is especially useful for things like navigation bars and floating chat buttons.
 
-### Sticky Positioning
+### Sticky positioning
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:35]:
```suggestion
### Sticky positioning

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 30]
```diff
@@ -29,10 +29,10 @@ In the following example, we are using absolute positioning to display text over
...
 Sticky elements will act like normal elements until you scroll past them, then they start behaving like fixed elements. They are also not taken out of the normal flow of the document. It might sound confusing, but checking out the behavior of [this](https://codepen.io/theanam/pen/MPLBYy) example might clear things up for you. It's useful for things like section-headings. Remember being able to still see what category you're looking at while scrolling through a shop? This is how it's done!
 
 ### Assignment
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:38]:
```suggestion
### Assignment

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 31]
```diff
@@ -44,14 +44,14 @@ Sticky elements will act like normal elements until you scroll past them, then t
 4. Finally, [this article](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/) discusses the difference between fixed and sticky positioning, it's a great read to understand the difference better.
 </div>
 
-### Knowledge Check
+### Knowledge check
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:47]:
```suggestion
### Knowledge check

```

File: intermediate_html_css/intermediate_css_concepts/positioning.md [hunk 32]
```diff
@@ -44,14 +44,14 @@ Sticky elements will act like normal elements until you scroll past them, then t
...
 This section contains questions for you to check your understanding of this lesson. If you’re having trouble answering the questions below on your own, review the material above to find the answer.
 
 - [What is the difference between static and relative positioning?](#static-and-relative-positioning)
 - [What is absolute positioning useful for?](#absolute-positioning)
 - [What is the difference between fixed and sticky positioning?](https://www.kevinpowell.co/article/positition-fixed-vs-sticky/)
 
-### Additional Resources
+### Additional resources
```
Comment (ManonLef) [inline intermediate_html_css/intermediate_css_concepts/positioning.md:54]:
```suggestion
### Additional resources

```

**Review by ManonLef — CHANGES_REQUESTED** [review]
Thanks so much again for this speedy review of so many lessons. 

I left a whole lot of suggestions for a newline after headers. This is not something that was part of the issue so I didn't expect you to fix that. If you don't see issues in my edits, please commit them (iyou don't have to do so manually if you click `commit suggestion`)

I also edited one or two headers with a question mark since they were questions. Same thing; commit those if you agree with them. But do let me know if you believe there's reason not to do this.

**Review by ManonLef — APPROVED** [review]

**Comment by ManonLef** [comment]
Wow that was a lot of clicking. I'm sorry! 

Thanks so much for doing this. It was a big section and a huge help!

**Comment by amc131** [comment]
Thank you for the opportunity! Always happy to give back to TOP

**Pull request merged**